docker/start-local-environment.sh

```diff
@@ -11,7 +11,7 @@ docker-compose up -d ps_db mhs-adaptor-mock activemq;
 
 cd ../db-connector
 ./gradlew update
-cd ../docker
+ cd ../docker
```

Comment:
Are those spaces here and before docker-compose commands necessary?

Reply:
Artefact of me tweaking them, removed